pvsite_datamodel/write/user_and_site.py

```diff
@@ -141,6 +144,9 @@ def create_site(
         dno = get_dno(latitude=latitude, longitude=longitude)
         dno = json.dumps(dno)
 
+    if owner is None:
+        owner = "test_owner"
```

Comment:
please leave it as None, so remove this bit

Reply:
if owner is None, I would actually get the user from user_uuid and set that as the owner